websites/B/backpack.tf/dist/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "service": "Backpack.tf",
+  "author": {
+    "name": "a＠a",
+    "id": "405028899798384640"
+  },
+  "category": "other",
+  "logo": "https://i.imgur.com/iHCD7nN.png",
+  "thumbnail": "https://i.imgur.com/uQCICM4.png",
+  "version": "1.0.0",
+  "color": "#757575",
+  "tags": [
+    "website"
+  ],
```

Comment:
weird tags :/

Reply:
I should have changed the metadata file at last, maybe that's why there are some things are not good enough